### PR TITLE
DTFile use V3 version as default

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: tiflash-upgrade-guide.md [hunk 1]
```diff
@@ -82,8 +82,10 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
 
 从 v6.1 升级至 v6.2 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考[从 v5.x 或 v6.0 升级至 v6.2](#从-v5x-或-v60-升级至-v62) 中关于 PageStorage 的描述。
 
-## 从 v6.x 或 v7.x 升级至 v7.3，并且设置了 `storage.format_version = 5`
+## 从 v6.x 或 v7.x 升级至 v7.3，并且设置了 `storage.format_version = 5`，或者升级至 v7.4
```
Comment (qiancai) [inline tiflash-upgrade-guide.md:85]:
```suggestion
## 从 v6.x 或 v7.x 升级至 v7.3 或者以上版本
```

Comment (qiancai) [reply tiflash-upgrade-guide.md:85]: 因为下一段文字已经提到 设置 `storage.format_version = 5`，所以这里可以简化下

File: tiflash-upgrade-guide.md [hunk 2]
```diff
@@ -82,8 +82,10 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
 
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+从 v7.4 开始，TiFlash 的 DTFile 默认版本为 V3。
+
+在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 或者升级到 v7.4后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
```
Comment (qiancai) [inline tiflash-upgrade-guide.md:91]:
```suggestion
在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 或者升级到 v7.4 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
```

File: tiflash-upgrade-guide.md [hunk 3]
```diff
@@ -82,8 +82,10 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
 从 v6.1 升级至 v6.2 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考[从 v5.x 或 v6.0 升级至 v6.2](#从-v5x-或-v60-升级至-v62) 中关于 PageStorage 的描述。
 
-## 从 v6.x 或 v7.x 升级至 v7.3，并且设置了 `storage.format_version = 5`
+## 从 v6.x 或 v7.x 升级至 v7.3，并且设置了 `storage.format_version = 5`，或者升级至 v7.4
 
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+从 v7.4 开始，TiFlash 的 DTFile 默认版本为 V3。
```
Comment (qiancai) [inline tiflash-upgrade-guide.md:89]: 7.4 中这个功能是 GA 还是实验特性呢

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -68,7 +68,7 @@ delta_index_cache_size = 0
     ## * format_version = 2 v6.0.0 以前版本的默认文件格式
     ## * format_version = 3 v6.0.0 及 v6.1.x 版本的默认文件格式，具有更完善的检验功能
     ## * format_version = 4 v6.2.0 及以后版本的默认文件格式，优化了写放大问题，同时减少了后台线程消耗
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:70]:
```suggestion
    ## * format_version = 4 v7.3.0 及以前版本的默认文件格式，优化了写放大问题，同时减少了后台线程消耗
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -68,7 +68,7 @@ delta_index_cache_size = 0
...
-    ## * format_version = 5 从 v7.3.0 开始引入的文件格式（非 v7.3.0 默认格式），该格式可以合并小文件从而减少了物理文件数量。注意该格式目前为实验特性，不建议在生产环境中使用。
+    ## * format_version = 5 从 v7.3.0 开始引入的文件格式（自v7.4.0 为默认格式），该格式可以合并小文件从而减少了物理文件数量。
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:71]:
```suggestion
    ## * format_version = 5 从 v7.3.0 开始引入的文件格式（v7.4.0 及以后版本的默认文件格式），该格式可以合并小文件从而减少了物理文件数量。
```

File: tiflash-upgrade-guide.md (hunk 3 above)
Comment (hongyunyan) [reply tiflash-upgrade-guide.md:89]: 是 GA 了的

File: tiflash-upgrade-guide.md [hunk 6]
```diff
@@ -82,8 +82,10 @@ TiFlash 在 v6.2.0 将数据格式升级到 V3 版本，因此，从 v5.x 或 v6
...
 从 v6.1 升级至 v6.2 时，需要注意 PageStorage 变更数据版本带来的影响。具体请参考[从 v5.x 或 v6.0 升级至 v6.2](#从-v5x-或-v60-升级至-v62) 中关于 PageStorage 的描述。
 
-## 从 v6.x 或 v7.x 升级至 v7.3，并且设置了 `storage.format_version = 5`
+## 从 v6.x 或 v7.x 升级至 v7.3 或以上版本
 
 从 v7.3 开始，TiFlash 支持新的 DTFile 版本 V3 (实验特性），可以将多个小文件合并成一个大文件，减少文件数量。DTFile 在 v7.3 的默认版本是 V2，如需使用 V3，可通过 [TiFlash 配置参数](/tiflash/tiflash-configuration.md) `storage.format_version = 5` 来设置。设置后，TiFlash 仍可以读 V2 版本的 DTFile，并且在后续的数据整理 (Compaction) 中会将这些 V2 版本的 DMFile 逐步重新写为 V3 版本的 DTFile。
 
-在 TiFlash 升级到 v7.3 并且使用了 V3 版本的 DTFile 后，如需回退到之前的 TiFlash 版本，可以通过 DTTool 离线将 DTFile 重新写回 V2 版本，详见 [DTTool 迁移工具](/tiflash/tiflash-command-line-flags.md#dttool-migrate)。
\ No newline at end of file
+从 v7.4 开始，TiFlash 的 DTFile 默认版本为 V3。
```
Comment (qiancai) [inline tiflash-upgrade-guide.md:89]:
```suggestion
从 v7.4 开始，DTFile 版本 V3 正式 GA，成为 TiFlash DTFile 的默认版本。
```

File: tiflash/tiflash-configuration.md [hunk 7]
```diff
@@ -67,8 +67,8 @@ delta_index_cache_size = 0
     ## DTFile 储存文件格式
     ## * format_version = 2 v6.0.0 以前版本的默认文件格式
     ## * format_version = 3 v6.0.0 及 v6.1.x 版本的默认文件格式，具有更完善的检验功能
-    ## * format_version = 4 v6.2.0 及以后版本的默认文件格式，优化了写放大问题，同时减少了后台线程消耗
-    ## * format_version = 5 从 v7.3.0 开始引入的文件格式（非 v7.3.0 默认格式），该格式可以合并小文件从而减少了物理文件数量。注意该格式目前为实验特性，不建议在生产环境中使用。
+    ## * format_version = 4 v7.3.0 及以前版本的默认文件格式，优化了写放大问题，同时减少了后台线程消耗
+    ## * format_version = 5 v7.4.0 及以后版本的默认文件格式（从 v7.3.0 开始引入），该格式可以合并小文件从而减少了物理文件数量。
     # format_version = 4
```
Comment (hongyunyan) [inline tiflash/tiflash-configuration.md:72]:
```suggestion
    # format_version = 5
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: tiflash-upgrade-guide.md
Comment (qiancai) [inline tiflash-upgrade-guide.md:1]: 根据  https://github.com/pingcap/docs-cn/pull/14751，这里的两个改动又 revert 回去了

**Pull request merged**